jekyll/_cci2/ip-ranges.adoc

```diff
@@ -58,9 +62,9 @@ workflows:
 ----
 
 [#example-configuration-pipeline-parameters]
-== Example configuration file using IP ranges with pipeline parameters.
+=== Example configuration file using IP ranges with pipeline parameters.
 
-Use conditional logic to control when IP ranges is enabled with pipeline parameters.
+Use conditional logic to control when IP ranges is enabled with pipeline parameters. You can set the pipeline parameter `pipeline.parameters.ip_ranges` to `true` to enable IP ranges for the `build` job. FOr more information on triggering pipelines with pipeline parameters, see the xref:triggers-overview#[Trigger a pipeline] page.
```

Comment:
In that example, the parameter's name is `ip_ranges`. The notation `pipeline.parameters.ip_ranges` is how you refer to the parameter.

Typo: `FOr`.

Reply:
Just checking - are you suggesting a change to the paragraph here? I've fixed the FOr typo but from your comment it seems like I need to do something else?